src/Controls/src/Core/Platform/Android/MultiPageFragmentStateAdapter.cs

```diff
@@ -1,14 +1,16 @@
-#nullable disable
-using System.Diagnostics.CodeAnalysis;
+using System;
+using System.Collections.Generic;
+using System.Linq;
 using AndroidX.Fragment.App;
 using AndroidX.ViewPager2.Adapter;
 
 namespace Microsoft.Maui.Controls.Platform
 {
-	internal class MultiPageFragmentStateAdapter<[DynamicallyAccessedMembers(BindableProperty.DeclaringTypeMembers)] T> : FragmentStateAdapter where T : Page
+	internal class MultiPageFragmentStateAdapter<T> : FragmentStateAdapter where T : Page
```

Comment:
I am not sure we can remove that as it is used by the `MultiPage<T>` field - and that still uses the attribute. Maybe not needed, but we can confirm with linker things later.